apps/src/javalab/JavalabFileExplorer.jsx

```diff
@@ -0,0 +1,133 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import color from '@cdo/apps/util/color';
+import onClickOutside from 'react-onclickoutside';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+const styles = {
```

Comment:
[non-blocking comment] There's a lot of styles here. Consider making this a css file instead. Here's a PR that uses a css file: https://github.com/code-dot-org/code-dot-org/pull/34679/files